src/util/Actor.hpp

```diff
@@ -154,16 +159,20 @@ class Actor : virtual public Allocated
             handle_message(std::move(message));
         }
     }
+private:
+    // unused - remove?
```

Comment:
Yeah the same at me current version of opentxs. It's private and not used anywhere so I guess we can safely remove it :)

Reply:
Message handling comes next, as it is too costly in terms of cpu time and I see a potential race condition built in.